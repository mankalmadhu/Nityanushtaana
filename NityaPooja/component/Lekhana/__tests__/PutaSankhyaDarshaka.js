import 'react-native';
import React from 'react';
import ಪುಟಸ೦ಖ್ಯಾದರ್ಶಕ from '../PutaSankhyaDarshaka';

import Renderer from 'react-test-renderer';

test("ಪುಟಸ೦ಖ್ಯಾದರ್ಶಕ" , () => {

expect(ಪುಟಸ೦ಖ್ಯಾದರ್ಶಕ(3,20)).toMatchSnapshot();

});