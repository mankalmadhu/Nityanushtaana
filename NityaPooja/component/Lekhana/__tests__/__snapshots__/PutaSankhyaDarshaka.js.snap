// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಪುಟಸ೦ಖ್ಯಾದರ್ಶಕ 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#595959",
      "bottom": 0,
      "height": 30,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#ffffff",
        "fontSize": 16,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
    >
      4
    </Text>
    /
    20
  </Text>
</View>
`;
